demo/src/app/create-case-filters-consumer.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="site-wrapper">
+    <main id="content" role="main">
+        <h1 class="heading-large">Create Case Filters component</h1>
+
+        <div class="text">
+            <p> This component is used to determine the jurisdiction, case type and event when creating a case. The elements of the contract are:</p>
+            <ul class="list list-bullet text">
+                <li>Input fields:
+                    <ul class="list list-bullet text">
+                        <li>jurisdictions - a collection of jurisdictions - including nested casetypes and events</li>
```

Comment:
Can we mention the details about Jurisdiction object in the array. Specifically which properties are required for the component to function properly?

Reply:
Please recheck.